src/appservice/cli.ts

```diff
@@ -14,12 +14,12 @@ import { IConfig } from "./config/config";
 import { Task } from "matrix-protection-suite";
 
 /**
- * This file provides the entrypoint for the appservice mode for mjolnir.
+ * This file provides the entrypoint for the appservice mode for draupnir.
  * A registration file can be generated `ts-node src/appservice/cli.ts -r -u "http://host.docker.internal:9000"`
  * and the appservice can be started with `ts-node src/appservice/cli -p 9000 -c your-confg.yaml`.
  */
 const cli = new Cli({
-  registrationPath: "mjolnir-registration.yaml",
+  registrationPath: "draupnir-registration.yaml",
```

Comment:
This will surely break existing deployments?

Reply:
it wont affect the playbook at all.

Reply:
> Like im not sure who we would break by renaming the Reg file considering your explicitly passing the reg file location to the AS when running AS mode

Ok